Description: Fix resolved detection, add stub-resolv.conf support
 Fix resolved detection, the symlink target is usually relative to the
 root, such that in chroots the file points to a file inside the
 chroot. But keep absolute targets too, as these may have been in use
 with older version of systemd.  Add support for stub-resolv.conf
 detection.
Author: Dimitri John Ledkov <xnox@ubuntu.com>

--- network-manager-1.8.2.orig/src/dns/nm-dns-manager.c
+++ network-manager-1.8.2/src/dns/nm-dns-manager.c
@@ -1645,6 +1645,11 @@ static gboolean
 _resolvconf_resolved_managed (void)
 {
 	static const char *const RESOLVED_PATHS[] = {
+		"../run/systemd/resolve/stub-resolv.conf",
+		"../run/systemd/resolve/resolv.conf",
+		"../lib/systemd/resolv.conf",
+		"../usr/lib/systemd/resolv.conf",
+		"/run/systemd/resolve/stub-resolv.conf",
 		"/run/systemd/resolve/resolv.conf",
 		"/lib/systemd/resolv.conf",
 		"/usr/lib/systemd/resolv.conf",
